addons/bestja_requests/security/security.xml

```diff
@@ -0,0 +1,179 @@
+<openerp>
+    <data>
+        <record id="templates_own_read_rule" model="ir.rule">
+            <field name="name">Request templates can be viewed by coordinator and managers of organization which created them.</field>
+            <field name="model_id" ref="model_bestja_requests_template"/>
+            <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+            <field name="domain_force">[
+                '|',
+                    ('organization.coordinator', '=', user.id),
+                    ('organization.projects.manager', '=', user.id),
+            ]</field>
+            <field name="perm_read" eval="True"/>
+            <field name="perm_write" eval="False"/>
+            <field name="perm_create" eval="True"/>
```

Comment:
"Request templates can be viewed and *created* by coordinator and managers of organisations which *created* them." would sound stupid ;)